tests/tx_validation_test.go

```diff
@@ -524,6 +690,26 @@ func decreaseGasPriceMagma(txType types.TxType, values txValueMap, contract comm
 	return values, err
 }
 
+func valueTransferToEOAWithCode(txType types.TxType, values txValueMap, contract common.Address) (txValueMap, error) {
+	txType = toBasicType(txType)
+	if txType == types.TxTypeValueTransfer || txType == types.TxTypeValueTransferMemo {
+		values[types.TxValueKeyTo] = contract
+		return values, kerrors.ErrNotEOAWithoutCode
+	}
+
+	return values, nil
+}
+
+func keyUpdateFromEOAWithCode(txType types.TxType, values txValueMap, contract common.Address) (txValueMap, error) {
+	txType = toBasicType(txType)
+	if txType == types.TxTypeAccountUpdate {
+		values[types.TxValueKeyFrom] = contract
+		return values, kerrors.ErrNotEOAWithoutCode
+	}
+
+	return values, nil
+}
+
```

Comment:
Can we utlize `validate7702` instead?

Reply:
That seems difficult. These are functions that set `from` or `to` which cause invalidation in validate7702 when they are of certain types.

Reply:
If you can pass `statedb`, setting `contract` address can be moved outside for example,
```go
if isValid := validate7702(statedb, txType, contract,  eoaWithCode.Addr); isValid {
    // set `contract` to `values`
    ...
```
Unfortunately, the function `validate7702` is currently private function, so it cannot be called in `tests` package. Let me close this.